server/routes/projects.js

```diff
@@ -4,7 +4,7 @@ const { Project } = require("../models/Project");
 const { User } = require("../models/User");
 
 //get all projects
-router.get("/", (req, res) => {
+router.get("/allProjects", (req, res) => {
```

Comment:
I think we can keep simply '/projects'. So here simply '/'.

To avoid confusion between client side routes and server side routes, we could add '/api' prefix to both project and user server routes.